seal.go

```diff
@@ -135,6 +155,148 @@ type KeyCreationParams struct {
 	AuthKey *ecdsa.PrivateKey
 }
 
+// SealKeyToExternalTPMStorageKey seals the supplied disk encryption key to the TPM storage key associated with the supplied public
+// tpmKey. This creates an importable sealed key and is suitable in environments that don't have access to the TPM but do have
+// access to the public part of the TPM's storage primary key. The sealed key object and associated metadata that is required
+// during early boot in order to unseal the key again and unlock the associated encrypted volume is written to a file at the path
+// specified by keyPath.
+//
+// The tpmKey argument must correspond to the storage primary key on the target TPM, persisted at the standard handle.
+//
+// This function expects there to be no file at the specified path. If keyPath references a file that already exists, a wrapped
+// *os.PathError error will be returned with an underlying error of syscall.EEXIST. A wrapped *os.PathError error will be returned if
+// the file cannot be created and opened for writing.
+//
+// This function cannot create a sealed key that uses a PCR policy counter. The PCRPolicyCounterHandle field of the params argument
+// must be tpm2.HandleNull.
```

Comment:
what's the invalidation plan given this?

Reply:
There isn't really going to be a way to support this with these types of objects because we need to be able to create the counter in order to compute the PCR policy.